src/pages/resultsView/mutation/MutationMapper.tsx

```diff
@@ -119,26 +118,27 @@ export default class MutationMapper extends React.Component<IMutationMapperProps
                                     </span>
                                 </div>)
                             }
-                            <ResultsViewMutationTable
-                                studyId={this.props.studyId}
-                                studyToCancerType={this.props.studyToCancerType}
-                                discreteCNACache={this.props.discreteCNACache}
-                                oncoKbEvidenceCache={this.props.oncoKbEvidenceCache}
-                                pubMedCache={this.props.pubMedCache}
-                                cancerTypeCache={this.props.cancerTypeCache}
-                                mutationCountCache={this.props.mutationCountCache}
-                                dataStore={this.props.store.dataStore}
-                                myCancerGenomeData={this.props.myCancerGenomeData}
-                                hotspots={this.props.store.indexedHotspotData}
-                                cosmicData={this.props.store.cosmicData.result}
-                                oncoKbData={this.props.store.oncoKbData}
-                                civicGenes={this.props.store.civicGenes.result}
-                                civicVariants={this.props.store.civicVariants.result}
-                                enableOncoKb={this.props.config.showOncoKB}
-                                enableHotspot={this.props.config.showHotspot}
-                                enableMyCancerGenome={this.props.config.showMyCancerGenome}
-                                enableCivic={this.props.config.showCivic}
-                            />
+                            {!this.props.store.clinicalDataForSamples.isPending && (
```

Comment:
This may not be an ideal solution, since now we are waiting for the clinical data to complete before starting rendering the table. If we don't wait for the clinical data, then conditional initial visibility setting won't work.